app/models/manager_refresh/inventory/persister.rb

```diff
@@ -0,0 +1,95 @@
+class ManagerRefresh::Inventory::Persister
+  attr_reader :manager, :target, :collections
+
+  # @param manager [ManageIQ::Providers::BaseManager] A manager object
+  # @param target [Object] A refresh Target object
```

Comment:
yeah, I need to finish the targeted refresh to know what will come through here, so I was thinking Object for now, which is always true :-D